app/views/historic_appointments/past_chancellors.html.erb

```diff
@@ -15,6 +15,10 @@
       <h2 class="profiles">20th &amp; 21st centuries</h2>
       <ol>
         <li class="person person-excerpt"><div class="inner">
```

Comment:
Yeah, I know, it looks really weird. But I don't want to reformat the entire file :)